packages/app/pages/organization/course/add.tsx

```diff
@@ -212,7 +240,71 @@ export default function Add(): ReactElement {
                       <></>
                     )}
                   </Col>
+                  <Col xs={{ span: 24 }} sm={{ span: 12 }}>
+                    <Form.Item
+                      name="course-features"
+                      label="Course Features"
+                      tooltip="Enable or disable features for this course"
+                    >
+                      <Checkbox.Group className="w-full">
+                        <Row justify="start">
+                          <Col xs={{ span: 12 }} sm={{ span: 6 }}>
+                            <Checkbox
+                              value="chatBotEnabled"
+                              style={{ lineHeight: '32px' }}
+                            >
+                              Chatbot&nbsp;
+                              <Tooltip
+                                title={
+                                  'This feature allows students to ask questions to an AI chatbot that will answer their questions based on uploaded lab content.'
+                                }
+                              >
+                                <QuestionCircleOutlined
+                                  style={{ color: 'gray' }}
+                                />
+                              </Tooltip>
+                            </Checkbox>
+                          </Col>
+                          <Col xs={{ span: 12 }} sm={{ span: 6 }}>
+                            <Checkbox
+                              value="queueEnabled"
+                              style={{ lineHeight: '32px' }}
+                            >
+                              Queues&nbsp;
+                              <Tooltip
+                                title={
+                                  'This feature allows students to ask questions in a queue that can then be answered by the professor or a TA. Suitable for online, hybrid, and in-person office hours and labs.'
+                                }
+                              >
+                                <QuestionCircleOutlined
+                                  style={{ color: 'gray' }}
+                                />
+                              </Tooltip>
+                            </Checkbox>
+                          </Col>
+                          <Col xs={{ span: 24 }} sm={{ span: 12 }}>
+                            <Checkbox
+                              value="asyncQueueEnabled"
+                              style={{ lineHeight: '32px' }}
+                            >
+                              Asynchronous Question Centre&nbsp;
+                              <Tooltip
+                                title={
+                                  'This feature allows students to ask questions asynchronously (e.g. outside of office hours or labs) that can then be responded to by the professor. It also features automatic AI-generated answers based on uploaded course content.'
```

Comment:
Typo?

```
to by
```

Reply:
I think i might be technically correct but also you're right this reads weird. Gonna change, thanks